scripts/contiv-vpp-bug-report.sh

```diff
@@ -100,6 +100,12 @@ master_kubectl get services -o wide --all-namespaces > k8s-services.txt
 echo " - networkpolicy"
 # Don't show 'No resources found.' error on stderr.
 master_kubectl get networkpolicy -o wide --all-namespaces > k8s-policies.txt 2>/dev/null
+echo " - statefulsets"
+# Don't show 'No resources found.' error on stderr.
+master_kubectl get statefulset -o wide --all-namespaces > k8s-statefulsets.txt 2>/dev/null
```

Comment:
Can you redirect stderr to the log file as well (and while you're at it, change the commands above to do the same)?  This way any problems collecting the data will be logged for us.